### ISSUE TITLE
IDopen admin user error

### ISSUE THREAD

**Issue by okoliec** [AAROC/DevOps#244]
Hello @brucellino,
I have completed install of idp-ldap on my system (https://idp.unilag.edu.ng:8443/login.action).  I can login with the initial user i added but i am unable to login with the useradmin that has privileges to approve registered users. I don't know if am using the wrong user name. But i confirmed from the output of ldap.

Pls can I solve this problem.
root@ldapnew:~# ldapsearch -x -H ldap:/// -b cn=Administrator,ou=Group,dc=idp,ou=Services,dc=local
# extended LDIF
#
# LDAPv3
# base <cn=Administrator,ou=Group,dc=idp,ou=Services,dc=local> with scope subtree
# filter: (objectclass=*)
# requesting: ALL
#

# Administrator, Group, idp, Services, local
dn: cn=Administrator,ou=Group,dc=idp,ou=Services,dc=local
objectClass: groupOfUniqueNames
objectClass: top
cn: Administrator
uniqueMember: cn=useradmin,dc=local
uniqueMember: cn=okoliec,ou=People,dc=local
description: Users in this group have administrative privileges on this server

# search result
search: 2
result: 0 Success

# numResponses: 2
# numEntries: 1

I can login with the initial user i added but I am having issues

**Comment by okoliec** [comment]
@brucellino..
Am still stocked here. Any clues on how to proceed.

Cheers

**Comment by okoliec** [comment]
Dear All,
I am wondering if anyone has come across this problem and managed to resolve it. I just reinstalled my idp-ldap and everything seems fine except that i am unable to login with the user created by ansible to manage users.
Below is the file that created the user and password but each time i tried to login with that username i get an invalid username or password. Please could someone confirm if the username am using is correct

root@ldapnew:/tmp# less useradmin.ldif 
dn: cn=useradmin,dc=local
objectClass: simpleSecurityObject
objectClass: person
objectClass: top
cn: useradmin
sn: useradmin
userPassword: ************
description: Account used by external tools to manage the users (e.g. the IdP)

**Comment by brucellino** [comment]
@okoliec check in the /tmp directory - there is an ldif created by Ansible with the credentials of the manager. If it looks ok, it should have the password in it, along with user name and other connection parameters. Try to connect via the web interface with that.

If that doesn't work there's probably a firewall issue.

**Comment by okoliec** [comment]
@brucellino : I have checked. I can login with the test user.

I tried adding adding user from the command line. Although i can login with the new username and password but i noticed that account is locked. There is a link to re-active the account which when clicked on reports account successfully reactivated but on login on with that username the account still requests for reactivation.

I also attempted registration a user by completing the registration form. The user receives in mail in the email address supplied and when the user clicks on the mail he is told to wait for for authorization from the admin.

The problem is that i cannot login with the username used for authorizing users. I guess the username is useradmin but i get an error that its either an invalid user or a wrong password.
